geode-redis/src/main/java/org/apache/geode/redis/internal/executor/key/KeysExecutor.java

```diff
@@ -45,7 +47,10 @@ public RedisResponse executeCommand(Command command,
     try {
       pattern = GlobPattern.compile(glob);
     } catch (PatternSyntaxException e) {
-      return RedisResponse.error(ERROR_ILLEGAL_GLOB);
+      logger.debug(
```

Comment:
Instead of debug, it seems like this should be a warning. Also the log message should mention that it was a pattern given to the KEYS command. Maybe the last sentence could be changed to "KEYS will return an empty list."
Also it seems like you should have some quotes around the curlies for the e.getMessage() and a period after the closing quote.

Reply:
updated it!